milestone_2/src/app/blog/page.tsx

```diff
@@ -0,0 +1,31 @@
+import Image from 'next/image'
+import styles from './page.module.css'
+import PageTitle from "@/components/page-title";
+import BlogPreview from '@/components/blogPreview';
+import {blogs} from "@/app/blogData";
+import {Blog} from "@/types/blog"
+
+export default function Blog() {
+  return (
+    <div>
```

Comment:
this div isnt doing anything. You can delete it and have the `main` tag serve as the highest tag

Reply:
This is relevant to all your other components. Although it isn't breaking anything in your case, if these were smaller components with more complex styling, it could lead to unpredictable behaviors.